src/components/Button/index.js

```diff
@@ -66,8 +67,7 @@ export const Button = ({
     // eslint-disable-next-line react/button-has-type
     <Component
       css={css`
-        font-family: ${theme.typography.bodyFont};
-        font-size: 17px;
+        ${css(interfaceSmall(theme))};
```

Comment:
I imagine we'll want this at the bottom of the styles in case a user wants to add any extra styles to this, right? Otherwise our styles would always take precedence, I think.

Reply:
Nope! Our styles should take precedence here, and users should override only the type style attributes in their themes. If they want to change other attributes (like margins) they should do that  by using component properties, or by extending the component itself. (OR possibly changing the spacing variables, but I haven't even written those ones yet!

I reserve the right to change my mind on this once we're actually testing theming.